independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CO2 Signal",
+    "description": "Co2 Signal intent is to allow users to consume electricity when its carbon footprint is lowest. \n\nBy using electricity at the right time, your device favors low-carbon sources of electricity",
+    "version": "1.0",
+	"contact": {

```

Comment:
@paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
Sorted - not sure how I missed this